react-components/src/components/Panel/Image360HistoricalDetailsPanel.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Chip, Tooltip } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+
+export interface Image360HistoricalDetailsPanelProps {
+  historicalCount?: number;
+  onDetailsClick?: () => void;
+}
+
+export const Image360HistoricalDetailsPanel = ({historicalCount, onDetailsClick}: Image360HistoricalDetailsPanelProps) => {
```

Comment:
I assume this is just a placeholder. Eventually use a context to get a viewer instance right?

Reply:
Correct, these are placeholder 😄